### ISSUE TITLE
AttributeError: module 'ffmpeg' has no attribute 'FFmpegManager'

### ISSUE THREAD

**Issue by spvkgn** [axcore/tartube#186]
dpkg -i python3-tartube_2.2.7.deb
```
$ tartube 
Traceback (most recent call last):
  File "/usr/bin/tartube", line 83, in <module>
    app = mainapp.TartubeApp()
  File "/usr/lib/python3/dist-packages/tartube/mainapp.py", line 249, in __init__
    self.ffmpeg_manager_obj = ffmpeg.FFmpegManager(self)
AttributeError: module 'ffmpeg' has no attribute 'FFmpegManager'
```

**Comment by axcore** [comment]
I'm sorry, I don't know enough about dkpg to say why this occuring or how to fix it.

I just tried it on my Linux Mint box, on which it does work:

![mintbox](https://user-images.githubusercontent.com/37951926/98381604-02b0be80-2042-11eb-945a-ec96be921fba.png)

**Comment by dan9550** [comment]
Had the same thing on Arch Linux after installing from the AUR and still after having manually installed ffmpeg. Will update after i do some more digging.

**Comment by axcore** [comment]
Tartube's code includes a file called `ffmpeg.py`. I suspect that when the Tartube code tries to call it, it calls some other ffmpeg-related code instead.

There will be a new release shortly, I will rename the file so it doesn't conflict with anything else.

**Comment by candrapersada** [comment]
why there is no button for direct download to the latest version of FFmpeg / AVConv in system preferences
`Click Edit > System preferences... > youtube-dlc > FFmpeg / AVConv`?
![image](https://user-images.githubusercontent.com/36833664/99729481-8f9d5280-2aed-11eb-88f5-17d0b6b9c84c.png)

**Comment by axcore** [comment]
On MS Windows, you can click **Operations > Install FFmpeg**. But that is not allowed on Linux or MacOS.

So, this tab is for Linux and MacOS users. Perhaps it's useful for 0.1% of Windows users, but for 99.9% of them, it's useless.

**Comment by candrapersada** [comment]
there is a bug in youtube-dlc where you have to use the latest version of ffmpeg and for ubuntu ffmpeg is an old version which is out of date and the updates are slow what else for ubuntu lts.

**Comment by axcore** [comment]
> why there is no button for direct download to the latest version of FFmpeg / AVConv in system preferences

Tartube cannot download anything when the preference/edit/configuration/wizard windows are open.

**Comment by axcore** [comment]
> Tartube's code includes a file called `ffmpeg.py`. I suspect that when the Tartube code tries to use it, some other ffmpeg-related file is called instead.
> 
> There will be a new release shortly, I will rename the file so it doesn't conflict with anything else.

Fixed in v2.3.0. The new .DEB file should work without problems.
